rfhub2/cli/keywords_importer.py

```diff
@@ -21,6 +31,12 @@
 INIT_FILES = {"__init__.txt", "__init__.robot", "__init__.html", "__init__.tsv"}
 
 
+@dataclass
+class CollectionUpdateWithKeywords:
+    collection: CollectionUpdate
```

Comment:
Should it be declared this way:
```
class CollectionUpdateWithKeywords:
    data: Tuple[CollectionUpdate, List[KeywordUpdate]]

Reply:
Or should I update the way the functions does return items as instance of this class?

Reply:
> Or should I update the way the functions does return items as instance of this class?

this one would be better